source/code/include/playfab/PlayFabJsonHeaders.h

```diff
@@ -6,7 +6,10 @@
 
 #include <playfab/PlayFabPlatformMacros.h>
 
-#ifdef PLAYFAB_PLATFORM_LINUX
+#if defined(PLAYFAB_PLATFORM_SWITCH)
+#include <json/json.h>
```

Comment:
including this particular json.h file gave me issues before about a monolithic header file and that we should really only use the sub classes (like json/value or json/reader) to save on space.

Reply:
"monolithic header file" is a Unreal warning, not an XPlat one.